game/hud/src/services/session/layout.ts

```diff
@@ -86,20 +92,26 @@ export function saySomething(s: string) : LayoutAction {
   }
 }
 
+export function adjustWidgetPositions() {
+  return {
+    type: ADJUST_POSITIONS,
+  }
+}
 
 
-let defaultWidgets = () => {
+const defaultWidgets = (): any => {
```

Comment:
8.1 is talking about the use of arrow functions not the definition of functions via arrows. It does not allow it.

However JB has said that the existing conventions of of the codebase are king so it is allowed